src/sentry/hybridcloud/models/webhookpayload.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import datetime
+from typing import Self
+
+from django.db import models
+from django.http import HttpRequest
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import Model, control_silo_only_model, sane_repr
+from sentry.utils import json
+
+THE_PAST = datetime.datetime(2016, 8, 1, 0, 0, 0, 0, tzinfo=datetime.UTC)
+MAX_ATTEMPTS = 10
+BACKOFF_FACTOR = 10
+
+
+@control_silo_only_model
+class WebhookPayload(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    mailbox_name = models.CharField(null=False, blank=False)
+    region_name = models.CharField(null=False)
+
+    # Scheduling attributes
+    schedule_for = models.DateTimeField(default=THE_PAST, null=False)
+    attempts = models.IntegerField(default=0, null=False)
+
+    date_added = models.DateTimeField(default=timezone.now, null=False)
+
+    # payload attributes
+    request_method = models.CharField(null=False)
+    request_path = models.CharField(null=False)
+    request_headers = models.TextField()
+    request_body = models.TextField()
+
+    class Meta:
+        app_label = "hybridcloud"
+        db_table = "hybridcloud_webhookpayload"
+
+    __repr__ = sane_repr(
+        "mailbox_name", "region_name", "schedule_for", "attempts", "request_method", "request_path"
+    )
+
+    @classmethod
+    def create_from_request(
+        cls, region: str, provider: str, identifier: int, request: HttpRequest
+    ) -> Self:
+        return cls.objects.create(
+            mailbox_name=f"{provider}:{identifier}",
+            region_name=region,
+            request_method=request.method,
+            request_path=request.get_full_path(),
+            request_headers=json.dumps({k: v for k, v in request.headers.items()}),
+            request_body=request.body.decode(encoding="utf-8"),
+        )
+
+    def next_schedule(self) -> datetime.datetime:
+        # TODO - decide on a backoff schedule
+        backoff_delta = datetime.timedelta(seconds=(BACKOFF_FACTOR * self.attempts + 1) * 2)
```

Comment:
This is really simple logic and doesn't increase quickly enough. Another option I considered was doubling `schedule_for - date_added * 2`. This would require special casing of `THE_PAST`.